gapis/gfxapi/vulkan/replay.go

```diff
@@ -128,6 +128,40 @@ func (t *makeAttachementReadable) Transform(ctx context.Context, id atom.ID, a a
 			out.MutateAndWrite(ctx, id, newAtom)
 			return
 		}
+	} else if recreateImage, ok := a.(*RecreateImage); ok {
```

Comment:
Yes. I think it is better to keep the form unchanged here? I can change to type switch if you prefer that way.